fraud-detection-update-sample/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# List of code owners for this repo
+*       @tnc-br/TimberIDOwner
```

Comment:
you don't need this file here do you?  You should inherit this from the root's CODEOWNERS